staging/src/k8s.io/apiserver/pkg/server/secure_serving.go

```diff
@@ -60,6 +60,9 @@ func (s *SecureServingInfo) tlsConfig(stopCh <-chan struct{}) (*tls.Config, erro
 	if s.MinTLSVersion > 0 {
 		tlsConfig.MinVersion = s.MinTLSVersion
 	}
+	if s.MaxTLSVersion > 0 {
+		tlsConfig.MaxVersion = s.MaxTLSVersion
```

Comment:
It is tls 1.3 by default, unless you want to explicitly tell server not to use 1.3. I can hardly imagine for what case

Reply:
I hope I understand your comment. Are you suggesting a code change, or are you wondering why someone would not want TLS 1.3?

If it is the latter: In our case we have a FIPS validated cryptography library that was validated for TLS1.2, but not TLS1.3. In order to run in FIPS validated mode, we would need to disable TLS1.3 for our aggregated apiserver.
This is an edge-case, but I suspect our organization is not the only one with a crypto module only validated for TLS1.2.

FIPS validation is a requirement for many US and CA government IT projects.

For more on FIPS: https://csrc.nist.gov/publications/detail/fips/140/2/final

Reply:
If you want FIPS support, the Go FIPS compiler already provides it.  See [this](https://github.com/vmware-tanzu/pinniped/blob/main/internal/crypto/ptls/fips_strict.go) as example.  I do not support the addition of any such configuration.

Reply:
Very enlightening, thank you. I suppose that means we could cancel this PR.